mobile/media.html

```diff
@@ -59,6 +59,7 @@ <h2>Technologies in progress</h2>
         <ul data-feature="P2P connections and audio/video streams">
           <li><a data-featureid="p2p">Peer-to-peer connection</a> across devices,</li>
           <li><a data-featureid="mst-content-hint">Content Hints</a> allowing Web applications to advertise the type of media content that is being consumed (e.g. speech or music, movie or screencast) so that user agents may optimize encoding or processing parameters,</li>
+          <li><a data-featureid="webrtc-svc">Scalable Video Coding</a> allowing Web applications to picking between possible encoding configurations on an outgoing WebRTC video track, makeing providing video at different qualities to multiple destinations for the same video stream easier,</li>
```

Comment:
SVC is both easy to grasp and hard to explain in a few words, but I think it would still be worthwhile to clarify what it is. Here's an attempt:

```html
<li><a data-featureid="webrtc-svc">Scalable Video Coding</a> (SVC) allowing Web
applications to configure encoding parameters to leverage SVC (whereby subset video
streams can be derived from the larger video stream by dropping packets to reduce
bandwidth consumption), making providing video at different qualities to multiple
destinations with the same initial video stream easier,</li>
```

Reply:
Looks good to me. Updated.